GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -155,10 +155,13 @@ public sealed partial class FormCommit : GitModuleForm
         private readonly AsyncLoader _unstagedLoader = new AsyncLoader();
         private readonly bool _useFormCommitMessage = AppSettings.UseFormCommitMessage;
         private readonly CancellationTokenSequence _interactiveAddSequence = new CancellationTokenSequence();
+        private readonly CancellationTokenSequence _cancellationTokenSequence = new CancellationTokenSequence();
```

Comment:
💡 Maybe a more descriptive name for this, since it's associated with a specific operation?